### PR TITLE
Provide density for AccessBridge, enable a11y for Windows

### PR DESCRIPTION
**Pull request by VeselovAlex**
This fix will work after merging changes introduced in [JetBrainsRuntime/105](https://github.com/JetBrains/JetBrainsRuntime/pull/105) and upstreaming to OpenJDK

### CODE REVIEW

File: compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt [hunk 1]
```diff
@@ -521,7 +521,7 @@ internal expect fun pointerInputEvent(
 internal expect fun makeAccessibilityController(
     skiaBasedOwner: SkiaBasedOwner,
     component: PlatformComponent
-): AccessibilityController?
```
Comment (olonho) [inline compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt:524]: Seems ability to not have a11y controller makes sense to keep.

**Comment by olonho** [comment]
What will happen with current JDKs?

File: compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt (hunk 1 above)
Comment (VeselovAlex) [reply compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt:524]: Nullability was a part of #106, and I've just revert it

**Comment by VeselovAlex** [comment]
> What will happen with current JDKs?

Nothing, a11y bugs will be present until JDK update but it should not affect other parts of an application. However this is a common issue of Java applications

**Comment by VeselovAlex** [comment]
Superseded by #173

**Pull request closed without merging**